haystack/utils/deepsetcloud.py

```diff
@@ -5,6 +5,8 @@
 import time
 from typing import Any, Dict, Generator, List, Optional, Tuple, Union
 
+from haystack import Label, Document, Answer
```

Comment:
I think this import introduces a cyclic dependency causing all tests to fail. Could you please try `from haystack.schema import ...`?